docs/js-guide.md

```diff
@@ -33,17 +36,58 @@ Projects may use any ES language features which meet all of the following criter
 As an exception, [JSX](https://reactjs.org/docs/introducing-jsx.html) is used for
 creating UI components.
 
+### Module naming
+
+Modules should use lowercase, hyphen-separated names (`helpful-module.js`).
```

Comment:
Must the module's names have two parts (separated by hyphen)?

Reply:
Only if it has multiple words. If the module name is just one word then it doesn't need hyphens.

Reply:
Thanks for answering my question.